app/views/hud_reports/_view_filter.haml

```diff
@@ -0,0 +1,50 @@
+:ruby
+  if @report.present?
+    path_for_clear = path_for_report(@report)
+  else
```

Comment:
I think this half of the `if` can go away now.  The recent UI cleanup means show doesn't list other reports, only history and measures/questions.   So, actually we'll need to replace the top half of the if  in the other scenario.

Reply:
✅